wpilibc/src/main/native/cpp/Filesystem.cpp

```diff
@@ -0,0 +1,31 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2008-2018 FIRST. All Rights Reserved.                        */
```

Comment:
```suggestion
/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
```

Reply:
This was copied directly from other source files:
https://github.com/wpilibsuite/allwpilib/blob/master/wpilibc/src/main/native/include/frc/Timer.h